### PR TITLE
Run e2e tests

### PR DESCRIPTION
**Pull request by pierDipi**
Depends on https://github.com/openshift-knative/serverless-operator/pull/1838 :heavy_check_mark:

### CODE REVIEW

File: openshift/images.yaml [hunk 1]
```diff
@@ -1 +1,39 @@
-{}
+knative.dev/eventing-kafka-broker/control-plane/cmd/kafka-controller: registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-kafka-controller:knative-v1.8
```
Comment (matzew) [inline openshift/images.yaml:1]: was this "generated" ?

Comment (pierDipi) [reply openshift/images.yaml:1]: Yes, by
```
  --images-from eventing \
  --images-from eventing-kafka-broker
```

Comment (matzew) [reply openshift/images.yaml:1]: Wanna a bit of doc for this? 
(can be in a different PR)

File: openshift/install.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ function install_eventing_with_mesh() {
     export ON_CLUSTER_BUILDS=true
     export DOCKER_REPO_OVERRIDE=image-registry.openshift-image-registry.svc:5000/openshift-marketplace
 
-    make OPENSHIFT_CI="true" TRACING_BACKEND=zipkin install-strimzi install-kafka-with-mesh || return $?
+    make OPENSHIFT_CI="true" SCALE_UP=5 TRACING_BACKEND=zipkin images install-strimzi install-kafka-with-mesh || return $?
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply openshift/install.sh:12]: I noticed that using 4 (used by `install-kafka-with-mesh`) was having instability issues for KafkaSourceScale tests due to pods in pending state, so 5 makes tests in CI more reliable.

File: openshift/images.yaml (hunk 1 above)
Comment (pierDipi) [reply openshift/images.yaml:1]: Any suggestion?

If you type `-h` on generate there is doc that explains `--images-from`:
```
Usage of generate:
      --dockerfile-build-dir string           Dockerfiles output directory for build image relative to output flag (default "ci-operator/build-image")
      --dockerfile-dir string                 Dockerfiles output directory for project images relative to output flag (default "ci-operator/knative-images")
      --dockerfile-image-builder-fmt string   Dockerfile image builder format (default "registry.ci.openshift.org/openshift/release:golang-%s")
      --dockerfile-source-dir string          Dockerfiles output directory for source image relative to output flag (default "ci-operator/source-image")
      --dockerfile-test-dir string            Dockerfiles output directory for test images relative to output flag (default "ci-operator/knative-test-images")
      --excludes stringArray                  File or directory regex to exclude (default [.*k8s\.io.*,.*knative.dev/pkg/codegen.*])
      --generators string                     Generate something supported: [dockerfile]
      --images-from stringArray               Additional image references to be pulled from other midstream repositories matching the tag in project.yaml
      --images-from-url-format string         Additional images to be pulled from other midstream repositories matching the tag in project.yaml (default "https://raw.githubusercontent.com/openshift-knative/%s/%s/openshift/images.yaml")
      --includes stringArray                  File or directory regex to include (default [test/test_images.*,cmd.*])
      --output string                         Output directory (default "/home/pierdipi/redhat/knative/eventing-istio/openshift")
      --project-file string                   Project metadata file path (default "/home/pierdipi/redhat/knative/eventing-istio/openshift/project.yaml")
      --registry-image-fmt string             Container registry image format (default "registry.ci.openshift.org/openshift/%s:%s")
      --root-dir string                       Root directory to start scanning, default to current working directory (default "/home/pierdipi/redhat/knative/eventing-istio")
pflag: help requested
```

Comment (matzew) [reply openshift/images.yaml:1]: > Any suggestion?

I think perhaps just a single notice on the `README` about the command itself?

And perhaps same for the install dependencies? As said, can be totally on a new PR:

> If you type -h on generate there is doc that explains --images-from:

The `-h` is helpful, thanks~


I generally find it hard to come to a repo with zero info on the scripts inside of different folders, or are the two targets on the make file enough?

File: openshift/install.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (matzew) [reply openshift/install.sh:12]: Great! Thanks!

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**